app/controllers/users_controller.rb

```diff
@@ -31,6 +36,10 @@ def update_status
   end
 
   def index
+    unless current_user
+      redirect_to(new_user_session_path)
```

Comment:
This change should not be necessary if you would leave before_action :authorize this would not be necessary

Reply:
changed it back